### DISCUSSION TITLE
Playlists don't allow skipping through songs using the now-playing overlay

### DISCUSSION THREAD

**Discussion by reo-the-enby** [ppy/osu#23268] [Q&A]
### Type

Game behaviour

### Bug description

I remember this working before but now in a playlist I couldn't skip trough the song.

### Screenshots or videos

https://user-images.githubusercontent.com/81715029/233588152-7316a3f8-6e69-40f2-9817-b7a91b9564aa.mov

### Version

2023.419.0

### Logs

[database.log](https://github.com/ppy/osu/files/11293928/database.log)
[input.log](https://github.com/ppy/osu/files/11293929/input.log)
[network.log](https://github.com/ppy/osu/files/11293930/network.log)
[performance-audio.log](https://github.com/ppy/osu/files/11293932/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/11293933/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/11293934/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/11293935/performance-update.log)
[performance.log](https://github.com/ppy/osu/files/11293936/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11293937/runtime.log)

**Comment by frenzibyte** [comment]
In a playlist room, there'll always be a playlist item selected, the beatmap of that item will be played on loop. It wouldn't make sense to start playing other songs while having a specific beatmap selected.

**Reply by reo-the-enby** [reply]
I agree. I wasn't talking about switching the song but skipping in the song. This is just such a non issue tho, I just dislike that when you select the songs in a playlist they start from the beginning rather from the preview point.
